monai/apps/auto3dseg/bundle_gen.py

```diff
@@ -177,17 +177,14 @@ def _run_cmd(self, cmd: str, devices_info: str):
         Execute the training command with target devices information.
 
         """
-        try:
-            logger.info(f"Launching: {cmd}")
-            ps_environ = os.environ.copy()
-            if devices_info:
-                ps_environ["CUDA_VISIBLE_DEVICES"] = devices_info
-            normal_out = subprocess.run(cmd.split(), env=ps_environ, check=True, capture_output=True)
-            logger.info(repr(normal_out).replace("\\n", "\n").replace("\\t", "\t"))
-        except subprocess.CalledProcessError as e:
-            output = repr(e.stdout).replace("\\n", "\n").replace("\\t", "\t")
-            errors = repr(e.stderr).replace("\\n", "\n").replace("\\t", "\t")
-            raise RuntimeError(f"subprocess call error {e.returncode}: {errors}, {output}") from e
+
+        logger.info(f"Launching: {cmd}")
+        ps_environ = os.environ.copy()
+        if devices_info:
+            ps_environ["CUDA_VISIBLE_DEVICES"] = devices_info
+        normal_out = subprocess.run(cmd.split(), env=ps_environ, check=True)
+        logger.info(repr(normal_out).replace("\\n", "\n").replace("\\t", "\t"))
```

Comment:
Without `capture_output`, `normal_out` will be something like less-informative:

```
CompletedProcess(args=['python', 'msg.py'], returncode=0)
```

We probably do not need to `replace` anything or even the entire `logger.info`.

Reply:
thank you. I agree, I'll remove the logger.info line